ush/jediinc2fv3.py

```diff
@@ -57,22 +60,136 @@ def jedi_inc_to_fv3(FV3JEDIinc, FV3inc):
             for name, variable in ncin.variables.items():
                 if len(variable.dimensions) == 4:
                     dimsout = variable.dimensions[1:]
+                    dimsout4 = dimsout
+                elif len(variable.dimensions) == 3:
+                    dimsout = variable.dimensions[1:]
+                    dimsout3 = dimsout
                 else:
                     dimsout = variable.dimensions
+
                 if name in vardict:
+                    if name == 'delp':
+                        continue
+                    if name == 'DELP':
+                        continue
+                    if name == 'delz':
+                        continue
+                    if name == 'T':
+                        tinc = 'T'
+                    if name == 't':
```

Comment:
It doesn't look like`tinc` is initialized/set to something. Are 'T' and/or 't' always present?

Reply:
The atmospheric jedi analysis increment file should always contain a temperature increment as 'T' or 't'.   The scripting has been compacted into a single logical test which sets `tinc` to the appropriate temperature variable in the jedi analysis increment file.